packages/regl-worldview/src/stories/Cubes.stories.js

```diff
@@ -43,6 +44,52 @@ const instancedCameraState = {
   perspective: true,
 };
 
+class DynamicCubes extends React.Component<any, any> {
+  state = { cubeCount: 1, cubeId: -1 };
+  onContainerClick = (e, args) => {
+    this.setState({ cubeId: args.clickedObjectId || -1 });
+  };
+
+  render() {
+    const { range } = this.props;
+    const { cubeCount } = this.state;
+    const cubes = new Array(cubeCount).fill(0).map((_, i) => {
+      const marker = cube(range, i + 1);
+      const { position } = marker.pose;
+      const { x, y, z } = position;
+      return {
+        ...marker,
+        color: marker.id % 2 ? [1, 1, 0, 1] : [1, 0, 0, 1],
+        pose: {
+          orientation: { x: 0, y: 0, z: 0, w: 1 },
+          position: { x: x + i * 4, y: y + i * 4, z: z + i * 4 },
+        },
+      };
+    });
+
+    function getHitmapProps() {
```

Comment:
yah could do - since it's a storybook I wasn't really concerned much about perf..more so wanted the code more co-located for readability. what you think?